### PR TITLE
Index page now shows message if searching is disabled. 

### LINKED ISSUE #1893: Hide the "Find a Trace" screen when searchEnabled=false
In support of https://github.com/openzipkin/zipkin/pull/1876, we should hide the "find a trace" and also maybe change the default content to empty or otherwise when the config.json has searchEnabled=false

### PR DESCRIPTION
**Pull request by Logic-32**
Fixes #1893.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
sweet!

**Comment by Logic-32** [comment]
:)

File: zipkin-ui/static/traces.properties [hunk 1]
```diff
@@ -6,6 +6,7 @@ traces.duration = Duration (μs) >=
 traces.limit = Limit
 traces.findBtn = Find Traces
 traces.sort = Sort
+traces.searchDisabled = Searching has been disabled via the searchEnabled property.
```
Comment (codefromthecrypt) [inline zipkin-ui/static/traces.properties:9]: @MrGlaucus @gianarb @drolando can you contribute chinese and italian text for this?

**Comment by codefromthecrypt** [comment]
I will merge this and if someone complains re-nag about i18n

**Comment by drolando** [comment]
Where is that message supposed to appear? I'm running 2.5.2 with SEARCH_ENABLED=false but I don't see any difference from the normal UI

**Comment by Logic-32** [comment]
I'm not sure where all @adriancole  merged that change.  According to the tags on the merge it should be in 2.5.2 but 2.5.3 seems the most likely to have it.

If there, it would disable the "Find a Trace" page and show a big red box instead of the search controls.

**Comment by codefromthecrypt** [comment]
from latest it actually appears the property is set but not having the
desired effect...


ZIPKIN_UI_SEARCH_ENABLED=false java -jar zipkin.jar

curl -s localhost:9411/configprops|jq
'."zipkin.ui-zipkin.autoconfigure.ui.ZipkinUiProperties"'

*{*

  *"prefix"**: *"zipkin.ui"*,*

  *"properties"**: {*

    *"defaultLookback"**: *3600000*,*

    *"environment"**: *""*,*

    *"instrumented"**: *".*"*,*

    *"dependency"**: {*

      *"lowErrorRate"**: *0.5*,*

      *"highErrorRate"**: *0.75

*    },*

    *"basepath"**: *"/zipkin"*,*

    *"queryLimit"**: *10*,*

    *"logsUrl"**: null,*

    *"searchEnabled"**: *false

*  }*

*}*

**Comment by Logic-32** [comment]
This is kinda funny...
```
// config.js
export default function loadConfig() {
  return $.ajax('config.json', {
    type: 'GET',
    dataType: 'json'
  }).then(data => function config(key) {
    return data[key] || defaults[key];
  });
```

Since we set `searchEnabled` to false, it defaults to the default.  Because that's how OR statements work :\

**Pull request merged**